### PR TITLE
adjusting message model for chat

### PR DESCRIPTION
**Pull request by codiebeulaine**
@justinvdm ready for review

### CODE REVIEW

File: mentorship/mentor/serializers.py [hunk 1]
```diff
@@ -173,11 +173,19 @@ class Meta:
...
         model = Message
-        fields = ('id', 'time_sent', 'sender', 'recipient', 'content')
-        read_only_fields = ('sender', 'recipient')
+        fields = (
+            'id', 'time_sent', 'sender', 'recipient', 'content', 'direction',
+            'message_type')
+        read_only_fields = ('sender', 'recipient', 'direction', 'message_type')
+
+    def get_direction(self, obj):
+        if self.context['request'].user.pk is obj.sender.user.pk:
```
Comment (justinvdm) [inline mentorship/mentor/serializers.py]: I think it is better practise and less error-prone to use `==` for int comparisons, can we use `==` here?

Comment (codiebeulaine) [inline mentorship/mentor/serializers.py]: sure

**Comment by justinvdm** [comment]
🍰 👍 wtih!

Wondering if we need to split out the user and system out messages the stats section of the admin UI since it is still showing all sent messages (user and system), when people might just be expecting user. Can be another PR though.

**Comment by codiebeulaine** [comment]
@justinvdm i do have a ticket for admin clean up, will add that to the list

**Pull request merged**